apps/cowswap-frontend/src/modules/hooksStore/pure/AppliedHookItem/styled.tsx

```diff
@@ -212,3 +212,19 @@ export const SimulateFooter = styled.div`
     padding: 2px;
   }
 `
+
+export const Spinner = styled.div`
```

Comment:
Could the existing spinner be used here?

https://github.com/cowprotocol/cowswap/blob/59cbc17225b825ee17ba0a9f93cfe3176fb2ca39/apps/explorer/src/components/common/Spinner.tsx#L1-L0

Reply:
Oh my 🤦 

Please disregard, I knew we had it but didn't notice it was from the Explorer